docs/modules/operation/pages/provisioning/detectors/FtpDetector.adoc

```diff
@@ -0,0 +1,31 @@
+= FTP Service Detector
+
+The FtpDetector is able to validate ftp connection dial-up processes.
+The detector can test ftp server on multiple ports and specific login data.
```

Comment:
```suggestion
It finds FTP server on multiple ports and can use also login credentials for service detection.
```

Reply:
We detect a FTP service using the command channel (21/tcp), right?

Reply:
Yes. https://github.com/OpenNMS/opennms/blob/7e9f5759badaf5636ae487a72b605a2a89e3e13e/opennms-provision/opennms-detector-lineoriented/src/main/java/org/opennms/netmgt/provision/detector/simple/FtpDetector.java#L46